packages/js/product-editor/src/components/schedule-sale-pricing-block/editor.scss

```diff
@@ -0,0 +1,16 @@
+.wp-block-woocommerce-product-schedule-sale-pricing {
+	.components-toggle-control__label {
+		display: flex;
+		align-items: center;
+	}
+
+	.woocommerce-date-time-picker-control.has-error {
```

Comment:
Nice!  Optional, but we could move this into settings for the editor for all input fields with `has-error`.

Reply:
Good question.  This is probably worth investigating on its own.  We can push this off to a future issue since it's way outside the scope of this PR.